cra-config.yaml

```diff
@@ -14,3 +14,14 @@ CRA_TARGETS:
       TF_VAR_provider_visibility: "public"
       TF_VAR_kms_encryption_enabled: true
       TF_VAR_enable_collecting_failed_events: true
+  - CRA_TARGET: "solutions/security-enforced" # Target directory for CRA scan. If not provided, the CRA Scan will not be run.
+    CRA_IGNORE_RULES_FILE: "cra-tf-validate-ignore-rules.json" # CRA Ignore file to use. If not provided, it checks the repo root directory for `cra-tf-validate-ignore-rules.json`
+    PROFILE_ID: "fe96bd4d-9b37-40f2-b39f-a62760e326a3"         # SCC profile ID (currently set to 'IBM Cloud Framework for Financial Services' '1.7.0' profile).
+    CRA_ENVIRONMENT_VARIABLES:
+      TF_VAR_prefix: "test-se"
+      TF_VAR_existing_resource_group_name: "Default"
+      TF_VAR_existing_kms_instance_crn: "crn:v1:bluemix:public:hs-crypto:us-south:a/abac0df06b644a9cabc6e44f55b3880e:e6dce284-e80f-46e1-a3c1-830f7adff7a9::"
+      TF_VAR_kms_endpoint_url: "https://e6dce284-e80f-46e1-a3c1-830f7adff7a9.api.private.us-south.hs-crypto.appdomain.cloud"
+      TF_VAR_existing_cos_instance_crn: "crn:v1:bluemix:public:cloud-object-storage:global:a/abac0df06b644a9cabc6e44f55b3880e:855ed836-05ce-4f39-98fa-508774f29323::"
+      TF_VAR_cross_region_location: "us"
+      TF_VAR_provider_visibility: "public"
```

Comment:
@rajatagarwal-ibm This wont work because we dont expose `provider_visibility` in security-enforced. We have no way to run CRA against private endpoints currently. So we can only scan fully configurable for now